2025 Haunted House/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -18,13 +19,16 @@
  */
 public class RobotContainer {
   // PCM1 is floor 1 of haunted house, PCM2 is floor 2
-  private final PneumaticsControlModule pcm1 = new PneumaticsControlModule(0);
-  private final PneumaticsControlModule pcm2 = new PneumaticsControlModule(1);
+    private final PneumaticsControlModule pcm1 = new PneumaticsControlModule(0);
```

Comment:
Your style-guide is up to you all as a team, but I think the added spaces here is incorrect.

Reply:
We use indent by 4 spaces, so I think the added spaces are correct. The previous 2 lines are deleted because they use indent by 2 spaces I believe.